build-protobuf-lite-arm-none-eabi.sh

```diff
@@ -28,7 +28,7 @@ eval "dos2unix autogen.sh"
 eval "./autogen.sh"
 
 # This sets up the repo and generates the makefiles and compile switches for our target
-eval "./configure --disable-shared --host=arm-none-eabi CPPFLAGS='-mthumb -mlittle-endian -ggdb -mcpu=cortex-m3 -DGOOGLE_PROTOBUF_NO_RTTI -DGOOGLE_PROTOBUF_NO_STATIC_INITIALIZER -DGOOGLE_PROTOBUF_NO_THREAD_SAFETY' LDFLAGS='-mthumb -mcpu=cortex-m3 --specs=nosys.specs'"
+eval "./configure --disable-shared --host=arm-none-eabi CPPFLAGS='-std=c++11 -mthumb -mlittle-endian -ggdb -mcpu=cortex-m3 -DGOOGLE_PROTOBUF_NO_RTTI -DGOOGLE_PROTOBUF_NO_STATIC_INITIALIZER -DGOOGLE_PROTOBUF_NO_THREAD_SAFETY' LDFLAGS='-mthumb -mcpu=cortex-m3 --specs=nosys.specs'"
```

Comment:
Probably not, we can remove this, The reason why I dockerized/Linux'd this was because Cygwin refused to build and I figured WSL2/Linux/Docker would be easier to work with.